### PR TITLE
Soarian R4 CareTeam documentation

### PR DESCRIPTION
**Pull request by sankalp1**
Description
Soarian R4 CareTeam documentation
----

PR Checklist
----
- [ ] Screenshot(s) of changes attached before changes merged.
Careteam - 1
Careteam - 2
Careteam - 3
Careteam - 4
Careteam - 5

- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: layouts/soarian/r4/clinical.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<h1 id="clinical">Clinical</h1>
+
+<p>The Clinical category of HL7<sup>®</sup> FHIR<sup>®</sup> resources includes Summary, Diagnostics, Medications, Care Provision, Request and Response.</p>
```
Comment (HRNWriter) [inline layouts/soarian/r4/clinical.html:3]: Add a comma(,) After Request, that is, Request. and Response.
PRODOC-169058.

File: lib/resources/r4/soarian_care_team.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+  children:
+
+  - name: role
+    required: 'Yes'
+    type: List of CodeableConcept
+    description: Role of the participant in care team.
+    note: Required if participant is given.
+    binding:
+      description: Role of participant in careteam.
```
Comment (HRNWriter) [inline lib/resources/r4/soarian_care_team.yaml:51]: Make careteam two words: care team
PRODOC-169058.

**Review by HRNWriter — COMMENTED** [review]
See my two two comments with minor changes, just adding a comma and changing one instance of careteam to care team.

**Comment by sankalp1** [comment]
@ShrutiNHegde This is the old PR - https://github.com/cerner/fhir.cerner.com/tree/Soarian_Fhir_R4_CareTeam_Write 
I have created a new PR without changing anything. Could you please complete the review

**Review by vf056083 — APPROVED** [review]

**Review by stanlysampath — APPROVED** [review]

**Review by NeethuKP — APPROVED** [review]

**Comment by sankalp1** [comment]
CT1

CT2

CT3

**Pull request merged**